app/proguard.cfg

```diff
@@ -166,5 +166,28 @@
     public static final int define_*;
 }
 
+#From SpongyCastle, sshj dependency
+-keep class org.spongycastle.**
+-keep class org.bouncycastle.**
+
+#We are not using these anyway
+-dontwarn org.spongycastle.jce.provider.X509LDAPCertStoreSpi
+-dontwarn org.spongycastle.x509.util.LDAPStoreHelper
+-dontwarn org.spongycastle.cert.dane.fetcher.JndiDANEFetcherFactory*
+-dontwarn org.bouncycastle.jce.provider.X509LDAPCertStoreSpi
+-dontwarn org.bouncycastle.x509.util.LDAPStoreHelper
+-dontwarn org.bouncycastle.cert.dane.fetcher.JndiDANEFetcherFactory*
+
+#From sshj. We are not using GSSAPI to connect to SSH
+-dontwarn net.schmizz.sshj.userauth.method.AuthGssApiWithMic
+
+#Warning was at SSHClient.authGssApiWithMic, referencing javax.security.auth.login.LoginContext.
+#But we are not using it too
+-dontwarn net.schmizz.sshj.SSHClient
+
+#ignore test classes
+-dontwarn android.test.**
+-dontwarn org.junit.**
+
 # for DexGuard only
--keepresourcexmlelements manifest/application/meta-data@value=GlideModule
\ No newline at end of file
+#-keepresourcexmlelements manifest/application/meta-data@value=GlideModule
```

Comment:
Why comment this line?

Reply:
Had problem running from Android studio with this enabled.

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithAndroidGradleClassShrinkerForDebug'.
> ProGuard configuration parser error: /home/trancelove/git/AmazeFileManager/app/proguard.cfg line 193:0 missing EOF at '-keepresourcexmlelements'
```

And I suppose we are not using the paid Dexguard but the free Proguard when building releases? Anyway, will uncomment this and let you decide.